test/callback.tests.js

```diff
@@ -0,0 +1,327 @@
+const assert = require('chai').assert;
```

Comment:
These tests are pretty much the same as `test/callback_route_form_post.tests.js` I just changed the name and reformatted them a little so they're a little more conventional (so I can run them from my IDE)

And added the last 2 tests